app/templates/components/component-tree-item.hbs

```diff
@@ -21,10 +21,26 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag 
+    {{if (and (not @item.args.positional) @item.isComponent) "component-tree-item__bracket"}} 
+    {{if (and (not @item.args.positional) (not @item.hasChildren)) "component-tree-item__bracket--self-closing"}}
+    {{if (and @item.args.positional @item.isComponent) "component-tree-item-classic__bracket"}} 
+    {{if (and @item.args.positional (not @item.hasChildren)) "component-tree-item-classic__bracket--self-closing"}}"
   >
     {{#if @item.isComponent}}
-      {{classify @item.name}}
+      {{#if @item.args.positional}}
+        {{dasherize @item.name}}
```

Comment:
I think we don't need to do anything special here, since the names already came dasherized